frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -440,17 +440,40 @@ describe("collection permissions", () => {
         cy.signInAsAdmin();
       });
 
-      it.skip("shouldn't record history steps when there was no diff (metabase#1926)", () => {
+      it("shouldn't render revision history steps when there was no diff (metabase#1926)", () => {
         cy.signInAsAdmin();
         cy.createDashboard("foo").then(({ body }) => {
           visitAndEditDashboard(body.id);
         });
+
         // Save the dashboard without any changes made to it (TODO: we should probably disable "Save" button in the first place)
         saveDashboard();
-        // Take a look at the generated history - there shouldn't be anything other than "First revision" (dashboard created)
-        cy.icon("ellipsis").click();
-        cy.findByText("Revision history").click();
-        cy.findAllByRole("button", { name: "Revert" }).should("not.exist");
+        cy.icon("pencil").click();
+        saveDashboard();
+
+        openRevisionHistory();
+
+        const revisionHistoryEntries = cy
+          .findAllByText("Bobby Tables")
+          .parent();
```

Comment:
This is a nitpick, really.
There is already an aliasing mechanism in Cypress that does exactly this (and then some more*)
```javascript
cy.findAllByText("Bobby Tables")
  .closest("tr") // a personal preference that is just a tiny bit more explicit and conveys more context
  .as("revisionHistoryEntries");
```
You can then reuse it later in the test like this:
```javascript
cy.get("@revisionHistoryEntries").doWhateverWithIt(...);
```
* _Cannot find the source but I **think** Cypress 'caches' aliases and doesn't have to query the DOM again if the element is still in the DOM. This is insignificant for smaller tests, but it definitely is an interesting concept and speeds things up a little bit._

Reply:
Oh nice, thank you!